ding/example/complex_observation.py

```diff
@@ -0,0 +1,266 @@
+from typing import Union, Optional, List, Any, Tuple
+import collections
```

Comment:
rename this example to `ppo_with_complex_obs.py`

Reply:
fix